WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Views/MigrationHeaderView.swift

```diff
@@ -90,7 +84,8 @@ final class MigrationHeaderView: UIView {
         static let labelsSpacing: CGFloat = 20
 
         static let titleFont: UIFont = WPStyleGuide.fontForTextStyle(.largeTitle, fontWeight: .bold)
-        static let primaryDescriptionFont: UIFont = WPStyleGuide.fontForTextStyle(.title2, fontWeight: .regular)
+        static let primaryDescriptionFont: UIFont = WPStyleGuide.fontForTextStyle(.body, fontWeight: .regular)
```

Comment:
Good catch! I think the primary and secondary descriptions had different font sizes, I checked Figma again and now they have the same font size.

Reply:
👍